assets/cloudcontrol.sh

```diff
@@ -42,8 +44,6 @@ then
     markdown < /home/cloudcontrol/.deprecation
```

Comment:
Just a question: Why don't you use the full path here? Then you do not need to adjust the PATH variable.

Reply:
The Path variable needs to be adjusted anyway for the basic functionality of cloudcontrol because everything is stored in /home/cloudcontrol/bin. So I just needed to move it up.